Makefile

```diff
@@ -387,18 +383,12 @@ create-management-cluster: $(KUSTOMIZE) $(ENVSUBST)
 	kubectl wait --for=condition=Available --timeout=5m apiservice v1beta1.webhook.cert-manager.io
 
 	# Deploy CAPI
-	kubectl apply -f https://github.com/kubernetes-sigs/cluster-api/releases/download/v0.3.7/cluster-api-components.yaml
+	curl -sSL https://github.com/kubernetes-sigs/cluster-api/releases/download/v0.3.7/cluster-api-components.yaml | $(ENVSUBST) | kubectl apply -f -
```

Comment:
Now that envsubst is called, if ENV vars are set (EXP_MACHINE_POOL=true, etc) the flags will be set to true.

Reply:
fyi @detiber @randomvariable you probably need to do the same in CAPA/CAPV